src/models/Settings.js

```diff
@@ -287,7 +290,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
               dataAttr: dataAttr,
               className: 'social-auth-button ' + dataAttr,
               title: function () {
-                return Okta.loc('socialauth.' + type + '.label');
+                return idp.text || Okta.loc('socialauth.' + type + '.label');
```

Comment:
in your example, `type` could have space (am guessing allowing other characters). Would it still work when type has space?
I wonder if it'll make life easier to only allow characters and digits?

Reply:
Ah!  Yes, I established what the criteria would be...and then utterly failed to enforce them.  Good catch.

Reply:
Oh yes - I didn't enforce since I'm not using the type yet, so this also falls under the other question (re: wherefore type)